cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillLauncher.java

```diff
@@ -157,4 +166,70 @@ private void deletePod(PodInfo podInfo) {
       LOG.warn("Failed to delete pod {} with uid {}", podInfo.getName(), podInfo.getUid(), e);
     }
   }
+
+  /**
+   * Get instance id from the annotation. The annotation will have array of boolean
+   */
+  private InstanceInfo getInstanceInfo(PodInfo podInfo) throws Exception {
+    int instanceId = 0;
+    int instanceCount = 1;
+
+    if (podInfo.getOwnerReferences().stream().anyMatch(r -> "Job".equals(r.getKind()))) {
```

Comment:
In general, if there is a large block within an if, it's more readable to invert condition to remove a layer of nesting:

```
if (!...) {
  return new InstanceInfo(0, 1);
}
...
```